package.json

```diff
@@ -1,24 +1,29 @@
 {
-  "name": "Hello-Books-BE",
-  "version": "1.0.0",
-  "description": "A system that allows efficient, cost-effective management of our library",
-  "main": "index.js",
-  "scripts": {
-    "start": "node server/index.js"
-  },
-  "repository": "git@github.com:KvNGCzA/Hello-Books-BE.git",
-  "author": "KvNGCzA <akanmuoluwadamilare@yahoo.com>",
-  "license": "MIT",
-  "dependencies": {
-    "cors": "^2.8.5",
-    "dotenv": "^8.0.0",
-    "express": "^4.17.1",
-    "express-validator": "^5.3.1",
-    "morgan": "^1.9.1"
-  },
-  "devDependencies": {
-    "eslint": "^5.16.0",
-    "eslint-config-airbnb-base": "^13.1.0",
-    "eslint-plugin-import": "^2.17.3"
-  }
+
+    "name": "Hello-Books-BE",
+    "version": "1.0.0",
+    "description": "A system that allows efficient, cost-effective management of our library",
+    "main": "index.js",
+    "scripts": {
+        "start": "node server/index.js",
+        "startDev": "babel-node server/index.js"
```

Comment:
Please make this `start:dev` instead of `startDev`

Reply:
done